README.md

```diff
@@ -642,6 +643,22 @@ expect(submitButtons).toHaveLength(3) // expect 3 elements
 expect(submitButtons[0]).toBeInTheDOM()
 ```
 
+## `within` API
+
+Sometimes, there is no garauntee that the text, placeholder, or label you want to query is unique on the page. So you might want to explicity tell react-render-dom to get an element **only within** a particular section of the page. `within` is a helper function for this case.
+
+Example: To get the text 'hello' only within a section called 'messages', you could do:
+
+```javascript
+import {render, within} from 'react-testing-library'
```

Comment:
Could you change this example to work better in the `dom-testing-library` docs?

```js
import {within} from 'dom-testing-library'

const {getByText} = within(document.body.getElementById('messages'))
const helloMessage = getByText('hello')
```

Also, could you add a note that says:


```md
> Fun fact, this is actually just an alias to the `getQueriesForElement` function!
```